ui.content/src/main/content/jcr_root/conf/saas-aem-module/_sling_configs/com.valtech.aem.saas.api.caconfig.SearchConfiguration/.content.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:jcr="http://www.jcp.org/jcr/1.0"
+    xmlns:nt="http://www.jcp.org/jcr/nt/1.0"
+    jcr:primaryType="cq:Page">
+    <jcr:content
+        jcr:primaryType="cq:PageContent"
+        index="/pgweb">
```

Comment:
not pgweb please. sample config can be valtechweb or something. no customer code please